src/services/rest/IRestRequest.ts

```diff
@@ -0,0 +1,5 @@
+export interface IRestRequest {
+  jsonPayload: any;
+  cache: string;
+  urlSuffix: string;
+}
```

Comment:
I think defining `IRestRequest` is overkill as it's only used internally by `RestService.ts`.

Reply:
+1 but it makes things cleaner. Lets remove it later when everything is done